projects/kit/directives/highlight/highlight.options.ts

```diff
@@ -0,0 +1,15 @@
+import type {WritableSignal} from '@angular/core';
+import {signal} from '@angular/core';
+import {tuiCreateToken} from '@taiga-ui/cdk';
+
+const DEFAULT = {
+    highlightColor: 'var(--tui-service-selection-background)',
+} as const;
+
+export interface TuiHighlightOptions {
+    readonly highlightColor: WritableSignal<string>;
+}
+
+export const TUI_HIGHLIGHT_OPTIONS = tuiCreateToken<TuiHighlightOptions>({
+    highlightColor: signal(DEFAULT.highlightColor),
```

Comment:
Interesting ! 
Do we need signal Here ? May be `string` would suffice

Reply:
May be string would suffice!

Reply:
yes, string is suffice [2eeef48](https://github.com/taiga-family/taiga-ui/pull/8812/commits/2eeef4854b57b7fd898849aee9449b60c47f1388)